docs/index.md

```diff
@@ -0,0 +1,135 @@
+# napari-animation documentation
```

Comment:
I really don't like that this `docs/index.md` file is an almost-identical copy of the `README.md`. This is a recipe for updates/changes being made in one place but not the other, and eventually ending up with users/developers having only partial information (depending on whether they mostly look at the built docs website, or mostly look at the github repo home page).

Can we change this, either to point sphinx directly to the main README.md in the repository, or somehow have the contents actually copied across *during* the build process?

Reply:
Sure, I didn't know what to write here so just copied the README as a placeholder. I can do an include, but there are some sections that won't work, for example the admonitions (since github markdown and myst markdown don't agree on that syntax)

Reply:
>  1B) Have bad admonition formatting on the github page. Personally, I'd prioritize making the official website look good.

This would be my choice also.